app/src/main/java/com/arygm/quickfix/ui/navigation/NavigationActions.kt

```diff
@@ -17,12 +27,54 @@ object Screen {
   const val INFO = "Info Screen"
   const val PASSWORD = "Password Screen"
   const val HOME = "Home Screen"
+  const val ANNOUNCEMENT = "Announcement Screen"
+  const val ACTIVITY = "Activity Screen"
+  const val OTHER = "Other Screen"
+  const val CALENDAR = "Calendar Screen"
+  const val MAP = "Map Screen"
 }
 
 data class TopLevelDestination(val route: String, val icon: ImageVector, val textId: String)
 
 object TopLevelDestinations {
   val HOME = TopLevelDestination(route = Route.HOME, icon = Icons.Filled.Home, textId = "Home")
+  val ANNOUNCEMENT =
+      TopLevelDestination(
+          route = Route.ANNOUNCEMENT, icon = Icons.Filled.AddCircle, textId = "Announcement")
+  val ACTIVITY =
+      TopLevelDestination(route = Route.ACTIVITY, icon = Icons.Filled.Menu, textId = "Activity")
+  val OTHER =
+      TopLevelDestination(route = Route.OTHER, icon = Icons.Filled.MoreVert, textId = "Other")
+  val CALENDAR =
+      TopLevelDestination(
+          route = Route.CALENDAR, icon = Icons.Filled.DateRange, textId = "Calendar")
+  val MAP = TopLevelDestination(route = Route.MAP, icon = Icons.Filled.Place, textId = "Map")
+}
+
+val USER_TOP_LEVEL_DESTINATIONS =
+    listOf(
+        TopLevelDestinations.HOME,
+        TopLevelDestinations.ANNOUNCEMENT,
+        TopLevelDestinations.ACTIVITY,
+        TopLevelDestinations.OTHER)
+val WORKER_TOP_LEVEL_DESTINATIONS =
+    listOf(
+        TopLevelDestinations.HOME,
+        TopLevelDestinations.CALENDAR,
+        TopLevelDestinations.MAP,
+        TopLevelDestinations.ACTIVITY,
+        TopLevelDestinations.OTHER)
+
+fun getBottomBarId(route: String, isUser: Boolean): Int {
+  return when (route) {
+    Route.HOME -> 1
+    Route.ANNOUNCEMENT -> 2
+    Route.CALENDAR -> 2
+    Route.MAP -> 3
+    Route.ACTIVITY -> if (isUser) 3 else 4
```

Comment:
I couldn't find where this was used in your code, so I'm not completly sure what its purpose is. But a small suggestion for logic like this could be: 

Since your app's UI changes based on the type of user, you'll probably need either a AppState that remembers this or a UserViewModel. So user based logic could be entirely contained in this viewModel. You'd now call viewModel.getBottomBarId(route) or viewModel.getBottomBarType(route). 

Also it's best to use objects like enums instead of raw ints